act-rules-format.md

```diff
@@ -497,6 +523,65 @@ Background (optional) {#background}
 
 
+Implementations (optional) {#implementation}
+--------------------------------
+
+An ACT Rule <em class="rfc2119">may</em> contain a list of implementations that are consistent with that rule. An [=implementation=] is an accessibility test methodology or test tool that can be used to produce [=outcomes=] of an ACT Rule.
+
+An [=implementation=] consists of multiple [=implementation procedures=]. The [=Implementation procedures=] are not required to have a one-to-one mapping to an ACT Rule. Mapping of an implementation to an ACT Rule is done by determining the set of implementation procedures with the highest consistency and coverage of that rule. Multiple implementation procedures can combine to map to a single ACT Rule, or a single implementation procedure can map to multiple ACT rules simultaneously.
+
+For each implementation the following data <em class="rfc2119">may</em> be provided. If included, consistency and coverage <em class="rfc2119">must</em> be determined as defined in this section.
+
+- Name of implementation
+- Name of the vendor
+- [Consistency](#impl-consistency)
+- [Coverage](#impl-coverage) 
+- Version used in testing consistency and coverage
+
+### Consistency ### {#impl-consistency}
+
+Consistency says how well a set of [=implementation procedures=] are aligned with the intent of an ACT Rule. Consistency is determined by comparing the [=outcome=] that implementation procedures return on the test cases to the expected outcomes of the test cases. In determining consistency, the `passed` and `inapplicable` outcomes are interchangeable. This supports implementations that do not distinguish between `passed` and `inapplicable`, and allows implementation procedures to be scoped different from an ACT Rule.
+
+In addition to `passed`, `failed`, and `inapplicable`, implementations can report `cantTell` and `untested` as an [=outcome=]. If no outcome is given for a test case, it is considered `untested`.
+
+There are three levels of consistency; complete, partial, and minimal. Consistency levels are defined in the following way. The consistency of a set of implementations procedures is the first level in the following list that matches the definition.
+
+1. <dfn>Complete</dfn> implementations consider all aspects of a rule's applicability and expectations. For the [=outcomes=] reported for some set of [=implementation procedures=], all the following are true:
+
+    - All `failed` test cases are reported as `failed`, or `cantTell` by at least one procedure; and
+    - All `passed` and `inapplicable` test cases are reported as `passed`, or `inapplicable`, or `cantTell` by all procedures; and
+    - `passed` and `failed` test cases are not all reported as `cantTell` by all procedures.
```

Comment:
Wondering if
```suggestion
    - at least one `passed` or `failed` test case is not reported as `cantTell` by all procedures.
```
is easier to swallow 🤔 
(moving the negation away from the quantifier)